draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -1011,6 +1225,7 @@ for their contributions (some of which substantial) to this draft and to the ini
 
 * update reference to IETF Status List
 * Include Type Metadata
+* Include display and claim type metadata
```

Comment:
-04 has been released to the wild so this needs to move into a -05

Reply:
Thanks, fixed!